nixos/modules/services/system/earlyoom.nix

```
@@ -115,6 +113,11 @@ in
         [README](https://github.com/rfjakob/earlyoom#notifications) and
         [the man page](https://github.com/rfjakob/earlyoom/blob/master/MANPAGE.md#-n-pathtoscript)
         for details.
+
+        WARNING: earlyoom is running in a sandbox with ProtectSystem="strict"
+        by default, so filesystem write is also prohibited for the hook.
+        If you wan to change these protection rules, override the systemd
+        service via `systemd.services.earlyoom.serviceConfig.ProtectSystem`.
```

```suggestion
        WARNING: earlyoom is running in a sandbox with ProtectSystem="strict"
        by default, so filesystem write is also prohibited for the hook.
        If you want to change these protection rules, override the systemd
        service via `systemd.services.earlyoom.serviceConfig.ProtectSystem`.
```
This warning should also be in the changelog.

---

Fixed. I added a note to incompatibility section in case of users are already relying on weak sandboxing.